dipy/core/tests/test_optimize.py

```diff
@@ -57,7 +58,7 @@ def test_optimize_new_scipy():
     npt.assert_array_almost_equal(opt.xopt, np.array([0, 0, 0, 0.]))
 
 
-@npt.dec.skipif(not SCIPY_LESS_0_12)
+@pytest.mark.skipif(not SCIPY_LESS_0_12)
```

Comment:
As you can see [here](https://travis-ci.org/nipy/dipy/jobs/355024064#L5253),  `pytest` need the parameter `reason`. so you need to update it like this : 
`@pytest.mark.skipif(not SCIPY_LESS_0_12, reason="requires scipy < 0.12")`